### PR TITLE
Add 1989 tracks

### PR DESCRIPTION
**Pull request by ShamblesSM**
- All 1989 tracks now point to their Taylor's Version.
- Wildest Dreams and This Love's Taylor's Versions now also point to the album version.
- Latest Update is now set to 1989.

### CODE REVIEW

File: src/mappings/1989.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+            // original
+            "spotify:track:2ls70nUDfjzm1lSRDuKxmw",
+            // original deluxe 
+            "spotify:track:1u8c2t2Cy7UBoG4ArRcF5g",
+            // Gameday Country 
+            "spotify:track:3LB2qckfpZq7aybaoZKRa2",
+            // BMRRS 
+            "spotify:track:1kHEuJRasudLhjvnbfc4yS",
+            // BM #1's Vol.4
+            "spotify:track:1YYOx55cW5JfMe65zEHBdu",
```
Comment (Roshy10) [inline src/mappings/1989.js:33]: wrong song: welcome to new york?

File: src/mappings/1989.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+            // original deluxe 
+            "spotify:track:3nRmDz7qGCvsMS30rGGY0x",
+            // Rep Stadium Tour 
+            "spotify:track:3YLA3NgzFRRa7LiHMtFPby",
+            // BMRRS
+            "spotify:track:7aMftpNNJiQj1c2rllc8Dk",
+            // Pride Month 
+            "spotify:track:1YYOx55cW5JfMe65zEHBdu",
+            // BM #1's Vol.4
+            "spotify:track:5oID3Qj3tTCZEH9eo0snxm"
```
Comment (Roshy10) [inline src/mappings/1989.js:17]: wrong track: shake it off

File: src/mappings/1989.js [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+            // Women of Country
+            "spotify:track:5pJvzBpE1KSMxEgMg5Ft31",
+            // For Your Quarantine
+            "spotify:track:6PRELLjOe8RaRIcW0sHskx",
+            // Women of Big Machine
+            "spotify:track:4ZspJzYIxOztSc9hZmcLzH",
+        ],
+    },
+    // Shake It Off
+    {
```
Comment (Roshy10) [inline src/mappings/1989.js:95]: duplicate album

**Review by Roshy10 — CHANGES_REQUESTED** [review]
Thanks for the PR! It looks like there's a few mistakes in the mappings unfortunately, so I can't let this through. Feel free to make another PR if you find I've missed anything

**Comment by ShamblesSM** [comment]
darn, as I expected. i'll go fix those right now

**Comment by ShamblesSM** [comment]
oops, I just realized you made a commit yourself already, feel free to close this if you wish, glad I could help with the initial thing tho despite the mistakes

**Comment by melvinmcrn** [comment]
Hey @ShamblesSM , I just opened PR #98 which includes both updates from you and @Roshy10 .
Please feel free to review them too! :)

**Comment by Roshy10** [comment]
this is supercedded by PR#105, but thank you so much for your effort, I really appreciate it!

**Pull request closed without merging**